packages/web-client/app/components/card-pay/deposit-workflow/transaction-status/index.css

```diff
@@ -36,3 +36,11 @@
   align-items: center;
   justify-content: center;
 }
+
+.transaction-status__delay {
+  color: #707070;
```

Comment:
The issue calls for grey text but I couldn’t find an existing Boxel colour that was readable 🤔

Reply:
It's a bit confusing but our most commonly used gray is a purple variant... `var(--boxel-purple-500);`, or `var(--boxel-purple-400);` might be good?